scripts/tests/export.sh

```diff
@@ -211,4 +211,38 @@ EOF
 # This simply tests that this does not hang (#1945).
 timeout -k 11m 10m "$earthly" --ci --push --remote-cache earthly/test-cache:export-test-7 +test7
 
+# Test 8: Earthly LABELS
+echo ==== Running test 8 ====
+rm -rf /tmp/earthly-export-test-8
+"$frontend" rmi earthly-export-test-8a:test || true
+"$frontend" rmi earthly-export-test-8b:test || true
+
+mkdir /tmp/earthly-export-test-8
+cd /tmp/earthly-export-test-8
+cat >> Earthfile <<EOF
+VERSION 0.7
+
+test8:
+    FROM busybox:latest
+    RUN echo "hello my world" > /data
+    SAVE IMAGE earthly-export-test-8a:test
+    LABEL foo=bar
+    SAVE IMAGE earthly-export-test-8b:test
+EOF
+
+"$earthly" prune --reset
+"$earthly" +test8
+
+label_count=$("$frontend" inspect earthly-export-test-8a:test | jq .[].Config.Labels | grep -c dev.earthly.)
+if [ "$label_count" -ne "3" ]; then
+    echo "Expected 3 dev.earthly labels on first image"
```

Comment:
it's helpful to see the actual value for cases where it fails.
```suggestion
    echo "Expected 3 dev.earthly labels on first image; got $label_count"
```